src/sybild/sybild.proto

```diff
@@ -0,0 +1,181 @@
+// sybild.proto
+//
+// This protobuf file describes the types and RPC endpoints for accessing a
+// sybil database over a network.
+syntax = "proto3";
+package pb;
+
+import "github.com/golang/protobuf/ptypes/struct/struct.proto";
+
+// Sybil is an OLAP analytics store.
+service Sybil {
+  // Ingest inserts new data into a sybil dataset.
+  rpc Ingest (IngestRequest) returns (IngestResponse) {}
+  // Query retrieves results from sybil.
+  rpc Query (QueryRequest) returns (QueryResponse) {}
+  // ListTables lists the tables in a sybil database.
+  rpc ListTables (ListTablesRequest) returns (ListTablesResponse) {}
+  // GetTable gets details about a particular table.
+  rpc GetTable (GetTableRequest) returns (Table) {}
+}
```

Comment:
we likely want Expire or Trim samples, but can think about how to do that later. it will require the sybild service calling out to all the servers and then figuring out the timestamp to expire all samples before.

Reply:
I'll put in stubs for now and can flesh out later.